examples/partial_weight_exchange_example/client.py

```diff
@@ -111,11 +111,24 @@ def get_parameters(self, config: Config) -> NDArrays:
         return self.parameter_exchanger.push_parameters(self.model, self.initial_model, config=config)
 
     def set_parameters(self, parameters: NDArrays, config: Config) -> None:
-        # Sets the local model parameters transfered from the server using a parameter exchanger to coordinate how
-        # parameters are set
-        assert self.model is not None and self.parameter_exchanger is not None
-        self.parameter_exchanger.pull_parameters(parameters, self.model, config)
-        # stores the values of the model parameters at the beginning of each training round.
+        """
```

Comment:
Not a suggestion for change but just a comment: I think it would make sense to create a client class that performs partial weight exchange (using whichever weight selection criterion). Right now there's no intermediate class between `BasicClient` and the very specific client in this example. 

Link to ClickUp: https://app.clickup.com/t/8685zgact

Reply:
I think that's a great idea. I actually also created a ticket for this a few days ago 🙂:

https://app.clickup.com/t/8685xd7uu

I'll merge the two tickets now!